src/plays/color-palettes-generator/components/SingleColor.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import rgbToHex from './utils';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
```

Comment:
@AbhiPatel10 , Why we are importing this here? we already have this in `src/index.js`.